includes/sections-menu-common.php

```diff
@@ -53,22 +53,19 @@ public static function format_shortcode_output( $content ) {
 	function section_menus_display_default( $selector ) {
 		ob_start();
 	?>
-		<nav class="navbar navbar-toggleable-md navbar-light bg-primary sections-menu">
+		<nav class="navbar navbar-toggleable-md navbar-light bg-primary sections-menu" style="box-shadow: 0 2px 32px rgba(0,0,0,.5);">
```

Comment:
It is the only style that was not available with Athena and I didn't think it was worth creating a css file for one style.